Makefile

```diff
@@ -58,6 +58,10 @@ ldflags := $(strip $(ldflags))
 BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
 
 #### Command List ####
+# Make sure you are using latest nitro replayer library before make install:
```

Comment:
is it possible to call `scripts/build.sh` from here? From an external user perspective, i'd still like to be able to just run
```
make install
```
and have my seid binary created directly.
@codchen @LCyson is this somethign we can check in? otherwise every time we run make install there's going to be a git diff. Or do we want to add `*.dylib` to `.gitignore`?

Reply:
+1 is it possible to invoke the shell script to automatically build the binary
and think we already check in a .dylib here: https://github.com/sei-protocol/sei-chain/tree/master/x/nitro/replay, I think the best practice here is to update the .dylib artifact with sei release so that we don't need to worry the replay dylib version (if people download and compile the nitro-replayer from scratch)

Reply:
can we try copying the checked in .dylib to the correct path and see if `make install` works in this case, lmk hdyt of this @yzang2019 @philipsu522